app/javascript/admin/controllers/buffer_controller.js

```diff
@@ -36,12 +43,4 @@ export default class BufferController extends Controller {
       badge.remove();
     }
   }
-
-  get bufferUpdateId() {
-    return parseInt(this.data.get('id'), 10);
-  }
-
-  set bufferUpdateId(value) {
-    this.data.set('id', value);
-  }
```

Comment:
this it's not used, but in general we can now use the Values API to pass values from the HTML to the controller